### ISSUE TITLE
Xcode build error - Non-void function does not return a value in all control paths

### ISSUE THREAD

**Issue by westley-ta** [CleverTap/clevertap-flutter#162]
Hi CleverTap team, I’m having this issue when upgrading to the latest xcode 14.3

```
Semantic Issue (Xcode): Non-void function does not return a value in all control paths
../ios/Pods/CleverTap-iOS-SDK/CleverTapSDK/CTLocalDataStore.m:412:0
```

Xcode: Version 14.3 (14E222b)
Flutter: 2.10.5
CT: clevertap_plugin 1.6.0

**Comment by akashvercetti** [comment]
@westley-ta  we are aware of this issue steming from Xcode 14.3 and above. We will be releasing a fix in this week itself. In the meantime, could you please XCode 14.2?

**Comment by westley-ta** [comment]
Yes, I currently downgrade it to Xcode 14.2.
Thanks!

**Comment by agarwalrohit2503** [comment]
Did it got fixed @akashvercetti  ?

**Comment by akashvercetti** [comment]
We have released a fix for this in our latest release v1.6.1 https://github.com/CleverTap/clevertap-flutter/releases/tag/1.6.1 👍

**Comment by ybbond** [comment]
fixed in my side after upgraded to 1.6.1. need to do
- `flutter clean`
-  then `cd ios` to do `rm -r Pods/ Podfile.lock`
- finally `pod install --repo-update`
